debian/changelog

```diff
@@ -1,3 +1,17 @@
+chipsec (1.2.5-2.1) unstable; urgency=medium
```

Comment:
We haven't merged your previous commit yet. Could you merge both changelog entries into one?